mainapp/migrations/0012_contributors.py

```diff
@@ -19,6 +19,7 @@ class Migration(migrations.Migration):
                 ('phone', models.CharField(max_length=10)),
                 ('address', models.TextField()),
                 ('commodities', models.TextField(verbose_name='What you can contribute')),
+                ('blood_group', models.CharField(choices=[('A+', 'A+'),('A-', 'A-'),('AB+', 'AB+'),('O+', 'O+'),('B+', 'B+'),('AB-', 'AB-'),('O-', 'O-'),('B-', 'B-')],max_length=5)),
```

Comment:
@adityakuppa26 Please do not alter existing django migrations files. Since you have added a new column to the table, you need to run `python manage.py makemigration` to autogenerate a new migration file. After that you need to run `python manage.py migrate` to create the column inside the database. Please readup on Django migrations before implementing this.